src/components/ui/forms/form-with-dynamic-fields.js

```diff
@@ -53,7 +54,8 @@ export const FormWDynamicFields = ({ fields }) => {
   });
 
   const onSubmit = data => {
-    console.log(data);
+    // for testing purposes
+    //console.log(data);
```

Comment:
We can use env variables to keep this active in local dev but disabled once committed.